Maintainers_Guide.md

```diff
@@ -0,0 +1,90 @@
+# BIDS Maintainers Group Guide
+
+The BIDS Maintainers are a key group of members that ensure that the BIDS infrastructure is kept up.
+The maintainers have wide-ranging responsibilities that enable BIDS to continue to grow and succeed.
+All maintainers are listed in the [DECISION-MAKING document](DECISION-MAKING.md#definitions),
+hosted at the bids-specification repository.
+As furthermore detailed in the DECISION-MAKING document,
+maintainers have a set of rights that go beyond those of a contributor.
+
+The following guide draws heavily on a related [blog post](https://matthewrocklin.com/blog/2019/05/18/maintainer)
+on “The Role of a Maintainer” by Matthew Rocklin,
+which is a recommended reading in conjunction with this guide.
+
+## Why become a maintainer?
+
+As a BIDS maintainer you may get the chance to:
+
+* Learn to work as a team
+* Bring your expertise to the BIDS maintainers group and cover technical blind spot it may have
+* Improving your technical writing skills (for example documentation)
+* Learn to work with continuous integration and development
+* Advise and participate in the development of BIDS extensions that are most commonly associated with a publication
+
+## Responsibilities
+
+* Maintainers need to be loosely aware of the entire project
+  and use their knowledge to facilitate and initiate interactions
+  between different nodes of the project
+  and determine a reasonable and timely order for features to be added and issues to be resolved.
+* Maintainers direct other BIDS contributors in reviewing PRs,
+  writing clarifications to the specification, or other contributions.
+  Maintainers “dispatch” work teams.
+* Maintainers ensure that all contributions maintain a friendly and welcoming tone
+  to encourage productive conversations.
+* If no work team is suitable or available,
+  the final responsibility of getting the work done lies with the maintainers.
+* The development of each BIDS extension proposal should be “followed”
```

Comment:
Should this be formalized? That is, each BEP would get a single maintainer point-of-contact? Should this be added to https://github.com/bids-standard/bids-extensions?

Reply:
I agree this should be mentioned in the bids extensions and also possibly explicitly mentioned in our listing of BEP.

Reply:
I agree and I opened this issue to track it: https://github.com/bids-standard/bids-extensions/issues/34